build/charts/theia/provisioning/datasources/init.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 Antrea Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+source $THIS_DIR/migrate.sh
+source $THIS_DIR/create_table.sh
+
+migrate
+createTable
+setDataVersion
```

Comment:
`setDataVersion` seems to be called already at the end of `migrate`. why calling again?

Reply:
Migration will be skipped if there is no data schema created before. In this case, I still want to set data version after running `createTable` function.

Reply:
Then you can just set the version only at the end of the init process, so that it's executed regardless of whether the migration is skipped or not?

Reply:
I am thinking about a case that the migration is finished but the Pod crashes during `createTable`. In this way I think we may want to have the version updated, as we do not want to the migration part to run again after the recover. But this might be a rare case and might only happen in development when we have something wrong in the `createTable`. Please tell me if you think it is safe to ignore this case.